generator/common-pipeline_test.go

```diff
@@ -0,0 +1,60 @@
+package generator
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
can we avoid using this additional package?
it looks like we can replace it with simple if/else constructs + eg, [t.Fatalf()](https://pkg.go.dev/testing#T.Fatalf)  instead of using require.ErrorContains() and require.Equal()?

Reply:
Addressing the code modifications